doc/api/process.md

```diff
@@ -2102,6 +2102,43 @@ false
 
 See the [TTY][] documentation for more information.
 
+## process.triggerFatalException(error)
```

Comment:
The second argument is not documented here. It's `fromPromise`.

Reply:
@BridgeAR I realized that yesterday... although I'm not really sure what it does? Does it only switch the `type` in `_fatalException`?